File: backend/src/main/java/isd/internship/ala/models/Role.java
package isd.internship.ala.models;

import javax.persistence.*;
import java.time.LocalDate;

@Entity
@Table(name = "roles")
public class Role {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ROLE_ID")
    private Integer id;

    @Column(name="ROLE")
    private String role;

    public Role() {
    }

    public Role(String role) {
        this.role = role;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getRole() {
        return role;
    }

    public void setRole(String role) {
        this.role = role;
    }
}

File: AngularDemo/src/app/role.ts


export class Role {

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get role(): string {
    return this._role;
  }

  set role(value: string) {
    this._role = value;
  }

  private _id: number;
  private _role: string;

  // public getRole(): string {
  //   return this._role;
  // }
  // public setRole(value: string): void {
  //   this._role = value;
  // }
}
